.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,15 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+# Command info
 
-**Version of the command being documented (if known):**
+- **Version of the command being documented:** `latest`
+- **Brief description of the command being documented:** unspecified
+<!-- Replace `latest` with a concreete version when you know it exactly. -->
+
+- [ ] New command pages or translations are added. <!-- Check if you create a documentation page to tldr-pages. -->
+- [ ] All command pages are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
+- [ ] All command pages have 8 or fewer examples.
+- [ ] All command page descriptions have links to documentation or a homepage.
+- [ ] All command pages follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines). <!-- Check when you are sure in it. -->
+- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message). <!-- Check when you are sure in it. -->
```

Comment:
```suggestion
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message). <!-- Check when you are sure in it. -->
```

I think the comment should be removed, it's implied that they should check. (It's the point of having it here, and why it's linked here.)